lib/broadway_rabbitmq/producer.ex

```diff
@@ -19,6 +21,9 @@ defmodule BroadwayRabbitmq.Producer do
       {:ok, queue_name, config} ->
         send(self(), :connect)
 
+        prefetch_count = config[:qos][:prefetch_count]
+        options = [buffer_size: prefetch_count * 2]
```

Comment:
Just in case, let's go with 5:

```suggestion
        options = [buffer_size: prefetch_count * 5]
```

It shouldn't matter really... but helps us to be a bit safer in case of constant disconnects.